aws/community-infra/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "environment" {}
```

Comment:
let's add some descriptions here and describe the vars

Reply:
done